pkg/apis/pipeline/v1beta1/task_validation_test.go

```diff
@@ -1524,11 +1579,11 @@ func TestIncompatibleAPIVersions(t *testing.T) {
 			t.Run(testName, func(t *testing.T) {
 				ts := tt.spec
 				ctx := context.Background()
+				ts.SetDefaults(ctx)
 				if version == "alpha" {
-					ctx = config.EnableAlphaAPIFields(ctx)
```

Comment:
Do we have any specific reason to move `ts.SetDefaults(ctx)` before the alpha condition block?